frontend/cypress/integration/projects/notifications.spec.ts

```diff
@@ -0,0 +1,139 @@
+/// <reference types="cypress" />
+
+const ENTERPRISE = Boolean(Cypress.env('ENTERPRISE'));
+const randomId = String(Math.random()).split('.')[1];
+const featureToggleName = `unleash-e2e-${randomId}`;
+const baseUrl = Cypress.config().baseUrl;
+let strategyId = '';
+const userIds: any[] = [];
+const userEmails: any[] = [];
+const userName = `user-e2e-${randomId}`;
+const projectName = `project-e2e-${randomId}`;
+const password = 'unleash4All';
+
+// Disable the prod guard modal by marking it as seen.
+const disableFeatureStrategiesProdGuard = () => {
+    localStorage.setItem(
+        'useFeatureStrategyProdGuardSettings:v2',
+        JSON.stringify({ hide: true })
+    );
+};
+
+// Disable all active splash pages by visiting them.
+const disableActiveSplashScreens = () => {
+    cy.visit(`/splash/operators`);
+};
+
+const createUsers = () => {
+    for (let i = 1; i <= 2; i++) {
+        const name = `${i}-${userName}`;
+        const email = `${name}@test.com`;
+        cy.request('POST', `${baseUrl}/api/admin/user-admin`, {
+            name: name,
+            email: `${name}@test.com`,
+            username: `${name}@test.com`,
+            sendEmail: false,
+            password: password,
+            rootRole: i + 1,
+        })
+            .as(name)
+            .then(response => {
+                const id = response.body.id;
+                userIds.push(id);
+                userEmails.push({ email, password });
+            });
+    }
+};
+
+const addMembersToProject = () => {
+    cy.request(
+        'POST',
+        `${baseUrl}/api/admin/projects/${projectName}/role/5/access`,
```

Comment:
this role/5 seems to be arbitrary. Difficult to figure out for a test reader where it comes from